src/context/WalletProvider/local-wallet.ts

```diff
@@ -0,0 +1,39 @@
+import { KeyManager } from './config'
```

Comment:
this file doesn't match our standard naming convention. Please use `camelCase` naming.